tests/test_start.py

```diff
@@ -15,6 +15,7 @@ def get_new_windows(selenium, timeout=2):
 def test_click_appstore(selenium, url):
     selenium.get(url("apps/apps/home/start.ipynb"))
     with get_new_windows(selenium) as handles:
+        time.sleep(5)
```

Comment:
The sleep is not at the right spot. It should be after clicking on the puzzle piece to give aiidalab enough time to collect some apps and display them. You can see that none of the apps are shown in the appstore screenshot.